.eslintrc.json

```diff
@@ -5,31 +5,42 @@
         "browser": true,
         "mocha": true
     },
-    "extends": [
-        "xo/esnext",
-        "xo-react"
-    ],
+    "extends": ["airbnb"],
+    "plugins": ["jest"],
     "rules": {
-        "arrow-parens": ["error", "as-needed", {
-            "requireForBlockBody": true
-        }],
+        "arrow-parens": ["error", "as-needed", { "requireForBlockBody": true }],
+        "comma-dangle": ["error", "never"],
+        "import/extensions": ["error", "never", { "json": "always" }],
+        "import/no-extraneous-dependencies": "off",
+        "import/no-unresolved": ["error", { "ignore": ["frontend"] }],
         "indent": ["error", 4, {
             "SwitchCase": 1
         }],
-        "key-spacing": ["error", {
-            "align": "colon"
-        }],
+        "jsx-a11y/no-static-element-interactions": "off",
+        "key-spacing": "off",
+        "no-confusing-arrow": "off",
```

Comment:
why?

Reply:
[Here's the description](https://eslint.org/docs/rules/no-confusing-arrow) in case you haven't read it. I don't remember why I added this one in particular, but to me

```js
const x = a => 1 ? 2 : 3;
```

looks a lot nicer than

```js
const x = a => { return 1 ? 2 : 3; };
```

Arrow functions are really handy because of their simple syntax, and I hate clobbering that feature with ugliness just because it looks like `>=`

😛